webcomponents/custom.html

```diff
@@ -323,7 +493,10 @@
         }
     </style>
     <template>
-        <div tabindex="0" id="short_desc">
+        <div tabindex="0" id="short_desc" role="region" aria-labelledby="short_desc_heading">
```

Comment:
This looks incorrect.  The id should be summary_desc not short_desc and labelledby be summary_desc_heading.